packages/dashboard/src/pages/events.page.tsx

```diff
@@ -7,16 +9,30 @@ import { EventsTable } from "../components/eventsTable";
 import MainLayout from "../components/mainLayout";
 import { addInitialStateToProps } from "../lib/addInitialStateToProps";
 import { requestContext } from "../lib/requestContext";
-import { PropsWithInitialState } from "../lib/types";
+import { AppState, PropsWithInitialState } from "../lib/types";
 
 export const getServerSideProps: GetServerSideProps<PropsWithInitialState> =
-  requestContext(async (_ctx, dfContext) => ({
-    props: addInitialStateToProps({
-      dfContext,
-      props: {},
-      serverInitialState: {},
-    }),
-  }));
+  requestContext(async (_ctx, dfContext) => {
```

Comment:
Can you also be sure to make these changes to the `/dashboard/users/[id]` page, which also displays events?